js/marquee.js

```diff
@@ -0,0 +1,49 @@
+class Marquee {
+  constructor(element) {
+    this.container = element;
+    this.data;
+    this.container;
+  }
+
+  start() {
+    this.MarqueeData();
+  }
+
+  async MarqueeData() {
+    const url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/quotes/`;
```

Comment:
the API URL should be stored in the config file you have created in a baseURL variable, instead of rewriting it on every usage of fetch(). Remeber DRY principle. ("https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/")